### ISSUE TITLE
Cannot open Warp without clicking "stable" in package contents

### ISSUE THREAD

**Issue by tcmaraist** [warpdotdev/Warp#2238]
### Discord username (optional)

tofu_BahnMi

### Describe the bug

The only way I am able to open Warp is by opening the package contents in Finder/Applications, then double clicking "stable" under MacOS. This will open Wap along with Terminal, and Terminal is showing every command the Warp is running. I am running Warp on an M1 Mac, Ventura.

### To Reproduce

I'm not sure how to reproduce this on another machine. Warp is running fine on my other computer.

### Expected behaviour

I expect Warp to open whenever I click the icon. I've downloaded a couple of other popular terminal apps to see if they would have the same issue opening, but they opened just fine. 

### Operating System

MacOS

### OS Version

13.0.1

### Shell Version

3.2.57

### Warp Version

v0.2022.11.14.14.55.stable_01

### Additional context

This is a new M1 Macbook Pro. I used Migration assistant to transfer all my data from my previous machine, a 2020 Macbook air. I have uninstalled warp and all associated files multiple times and reinstalled via the link from warp.dev and homebrew. I have double checked all permissions that I can think of. The only way I am able to get Warp to open is by double clicking "stable" under MacOS in Warp's package contents.

### Does this block you from using Warp daily?

Yes

**Comment by dannyneira** [comment]
Hi Sorry about this, can you please try uninstalling, then reinstalling via brew, if the issue persists gather console logs and let us know.
https://docs.warp.dev/help/uninstalling-warp
https://docs.warp.dev/getting-started/getting-started-with-warp
https://docs.warp.dev/help/sending-us-feedback#gathering-warp-logs

**Comment by dannyneira** [comment]
Yeah so sorry about that, we've got a huge interest in the app so are focused on GitHub issues for now. I'll bring this up with the team and we will post any updates here if/when this is being worked on. it sounds like something may have gone wrong in the migration, so will make this difficult to reproduce ( esp since you have no logs ). Thanks for your patience.

**Comment by tcmaraist** [comment]
All good, I understand. I have tried clearing the cache and any files associated with Warp, but no luck. The only thing I can think is it might be a permissions issue, but I haven't been able to find which one specifically. What all permissions does Warp need?

**Comment by dannyneira** [comment]
Go with `full disk access` preferably or `files and folders` minimum.

**Comment by dannyneira** [comment]
@tcmaraist any luck with the above permissions?

**Comment by tcmaraist** [comment]
No luck. I might try reinstalling the OS at some point. I’ve also tried
running Warp in Rosetta. The only thing that gets it to open is double
clicking “stable” in package contents, but that also opens Terminal.app

Best,

Ty Maraist

On November 29, 2022, GitHub ***@***.***> wrote:
> @tcmaraist <https://github.com/tcmaraist> any luck with the above
> permissions?
>

—
Reply to this email directly, view it on GitHub
<https://github.com/warpdotdev/Warp/issues/2238#issuecomment-
1331367503>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AUH6EHUIGUE4KFVNHDEZ72LWKZ3ELANCNFSM6AAAAAASMAUQHE>.
You are receiving this because you were mentioned.Message ID:
***@***.***>

**Comment by dannyneira** [comment]
Just wanted to clarify here: 
Did you transfer Warp from another Mac to this new M1 mac or is this a fresh install? 
If the fresh install, was it from the DMG or Brew?
Would you be open to a troubleshooting session to go over the issue? https://calendly.com/danny-warp/troubleshoot

**Comment by zhouqicf** [comment]
> Terminal
"The only thing that gets it to open is double
clicking “stable” in package contents, but that also opens Terminal.app"

+1

**Comment by tcmaraist** [comment]
my issue was that warp was set to "read only"
